src/inmanta/moduletool.py

```diff
@@ -1132,5 +1132,6 @@ def get_setup_cfg(self, config_in: Optional[configparser.ConfigParser] = None) -
         config["options"]["zip_safe"] = "False"
         config["options"]["include_package_data"] = "True"
         config["options"]["packages"] = "find_namespace:"
+        config["options.packages.find"]["include"] = "inmanta_plugins"
```

Comment:
Otherwise submodules are not packaged.
```suggestion
        config["options.packages.find"]["include"] = "inmanta_plugins*"
```

Reply:
I would also add a test to verify that submodules are packaged as well. If this test doesn't exist yet.

Reply:
Have you tested this? I seem to recall I tried this first and it didn't work (but I could be wrong). I haven't tested submodules so it's very much possible that my suggestion doesn't work there.

Reply:
Yes, I did.